blockchain/blockdao.go

```diff
@@ -29,46 +29,22 @@ import (
 )
 
 const (
-	blockNS                             = "blocks"
-	blockHashHeightMappingNS            = "hash<->height"
-	blockTransferBlockMappingNS         = "transfer<->block"
-	blockVoteBlockMappingNS             = "vote<->block"
-	blockExecutionBlockMappingNS        = "execution<->block"
-	blockActionBlockMappingNS           = "action<->block"
-	blockActionReceiptMappingNS         = "action<->receipt"
-	blockAddressTransferMappingNS       = "address<->transfer"
-	blockAddressTransferCountMappingNS  = "address<->transfercount"
-	blockAddressVoteMappingNS           = "address<->vote"
-	blockAddressVoteCountMappingNS      = "address<->votecount"
-	blockAddressExecutionMappingNS      = "address<->execution"
-	blockAddressExecutionCountMappingNS = "address<->executioncount"
-	blockAddressActionMappingNS         = "address<->action"
-	blockAddressActionCountMappingNS    = "address<->actioncount"
-	receiptsNS                          = "receipts"
+	blockNS                          = "blk"
```

Comment:
which one is more impactful, namespace or prefix? Maybe name space does't matter?

Reply:
I guess prefix. I suspect boltdb impl doesn't put bucket name into every single tuple.